### ISSUE TITLE
Redactor - List (either) issue

### ISSUE THREAD

**Issue by TechCare-Aus** [osTicket/osTicket#5366]
### Prerequisites

* [ ] Can you reproduce the problem in a fresh installation of the "develop" branch?
* [X] Do you have any errors in the PHP error log, or javascript console?
* [X] Did you check the [osTicket forums](https://forum.osticket.com/)?
* [X] Did you [perform a cursory search](https://github.com/osTicket/osTicket/issues) to see if your bug or enhancement is already reported?

### Description

In both Chrome and Firefox if I type in a message or note and then click on a new line (pressing return a few times) click the list option (either) it will place all the text into whichever option of the list I choose.

I'm running the latest git version.

### Steps to Reproduce

1. Enter text into the message/note field
2. Click on the list function in redactor
3. Notice issue

**Expected behavior:** [What you expected to happen]

List forms AFTER text (or includes text highlighted)

**Actual behavior:** [What actually happened]

List forms for ALL text
![image](https://user-images.githubusercontent.com/2740134/73908424-a408b980-48e4-11ea-94dc-50044a9abef9.png)

### Versions

![image](https://user-images.githubusercontent.com/2740134/73908479-c69ad280-48e4-11ea-9f74-3de94b391f27.png)

**Comment by JediKev** [comment]
@TechCare-Aus 

I do not see the same issues, of course I'm using a Mac. Have you tried clearing all of your cache, etc.? Also have you tried setting up a Vanilla install of osTicket to see if that experiences the same issues? 

Cheers.

**Comment by TechCare-Aus** [comment]
@JediKev 
Brand spanking new install spun up just now same issues:

[10-Feb-2020 15:08:41 UTC] PHP Warning:  session_destroy(): Session callback expects true/false return value in /home/USERNAME/public_html/testing/scp/logout.php on line 30
[10-Feb-2020 15:08:41 UTC] PHP Warning:  session_destroy(): Session object destruction failed in /home/USERNAME/public_html/testing/scp/logout.php on line 30

![image](https://user-images.githubusercontent.com/2740134/74161850-a2c7eb80-4c5a-11ea-8f11-aadcb9fd44d1.png)

**Comment by JediKev** [comment]
@TechCare-Aus 

Have you tried a different computer? No matter what I try I cannot replicate this issue.  Below is what happens when I write text on many new lines and select the List option for the last line:
<img width="922" alt="Screen Shot 2020-02-10 at 09 12 11" src="https://user-images.githubusercontent.com/11823401/74162013-96ee1200-4be5-11ea-9de8-fd31e95bfb50.png">

Cheers.

**Comment by TechCare-Aus** [comment]
I apologies @JediKev I didn't read the title of this issue.. I thought we were talking about the session destroy issue I have open as well.

Can I remove the incorrect reply to this ??

Also with a new osTicket running on version 1.14.1 I do NOT have the issue. Hmmm. I've tried different browsers and running in incognito as well.

**Comment by JediKev** [comment]
@TechCare-Aus 

>Can I remove the incorrect reply to this ??

Yes, click the three dots on the reply and click Delete. If you can't lmk and I'll delete it.

>Also with a new osTicket running on version 1.14.1 I do NOT have the issue. Hmmm. I've tried different browsers and running in incognito as well.

No problem, I can close this then right?

Cheers.

**Comment by TechCare-Aus** [comment]
@JediKev Nope. Sorry.

New install updated to the latest Git. Then tested with the double spacing turned on works fine. Then tested with Single spacing turned on and issue presents itself.

Fresh install. Changed to PHP v7.3 just in case that was the problem too.

Looks to be a redactor issue perhaps?

**Comment by JediKev** [comment]
@TechCare-Aus 

Yes, tested this on their demo on their website and I can reproduce the same issue, looks to be an upstream Redactor issue.

Cheers.

**Comment by TechCare-Aus** [comment]
Ok brilliant.

**Comment by JediKev** [comment]
@TechCare-Aus 

So, revisiting this issue, I'm upgrading redactor (which will come soon) and I do believe they addressed this issue. I'm no longer able to replicate this using the latest codebase.

I will link the pull here when it gets pushed.

Cheers.

**Comment by TechCare-Aus** [comment]
@JediKev - fantastic. Looking forward to being able to use the single space and normal redactor again.
